Build/build.ps1

```diff
@@ -8,15 +8,17 @@ $all_ok = $True
 
 function Build-One {
     param(
-        $project
+        $project,
+        $qirGenerationFlag
```

Comment:
Minor PowerShell style suggestion, but it may be more natural to make this a switch parameter instead:

```suggestion
        [switch]
        $GenerateQir
```

Doing so, line 21 would then become:

```
/property:QirGeneration=($GenerateQir ? "true" : "false")
```

Similarly, lines 90–91 become:

```
$QirProjects `
    | ForEach-Object { Build-One $_ -GenerateQir }
```

Reply:
I'm not familiar with switch parameters, and I can't seem to get your suggestion to work properly. Could you please take another look at this?

Reply:
Sounds good, will do. It's also a minor comment at most; didn't mean to cause too much extra work, sorry.

Reply:
Fixed